app/code/Magento/CatalogRule/README.md

```diff
@@ -1 +1,68 @@
-Magento_CatalogRule module is responsible for one of the types of price rules in Magento. Catalog Rules are applied to products before they are added to the cart.
+# Magento_CatalogRule module
+
+**Magento_CatalogRule** module is responsible for one of the types of price rules in Magento 2.
+
+**Magento_CatalogRule** module applied rules to products before they are added to the cart.
+
+## Installation details
+
+The **Magento_CatalogRule** module can`t be removed, because have dependencies with the **Magento_CatalogRule** module.
```

Comment:
```suggestion
The **Magento_CatalogRule** module cannot be removed, because have dependencies with the **Magento_CatalogRule** module.
```

Reply:
See how the code coloring is off here on Github. A regular single quote will do here. Backticks are reserved for markdown. But we don't allow contractions in docs, so can't becomes cannot.

Reply:
@dobooth, thanks for the note. Changes were added.